### PR TITLE
Create JSON schema dumps for OGGBundle.

### PR DESCRIPTION
**Pull request by lukasgraf**
This is a basic implementation of JSON schema dumps that describe the content types that make up an OGGBundle.

@deiferni

### CODE REVIEW

File: docs/schema-dumps/oggbundle/documents.schema.json [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+                    "default": "<Aktuelles Datum>"
+                },
+                "receipt_date": {
+                    "type": "string",
+                    "title": "Eingangsdatum",
+                    "format": "date",
+                    "description": "Datum, an dem das Dokument \u00fcber den Korrespondenzweg angekommen ist",
+                    "_zope_schema_type": "Date"
+                },
+                "changeNote": {
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/documents.schema.json]: should no be here i think

File: docs/schema-dumps/oggbundle/documents.schema.json [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+                "review_state": {
+                    "type": "string"
+                },
+                "foreign_reference": {
+                    "type": "string",
+                    "title": "Fremdzeichen",
+                    "description": "Referenz auf das entsprechende Dossier des Absenders",
+                    "_zope_schema_type": "TextLine"
+                },
+                "thumbnail": {
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/documents.schema.json]: should not be here?

File: docs/schema-dumps/oggbundle/documents.schema.json [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+                    ]
+                },
+                "preserved_as_paper": {
+                    "type": "boolean",
+                    "title": "In Papierform aufbewahrt",
+                    "description": "In Papierform aufbewahrt",
+                    "_zope_schema_type": "Bool",
+                    "default": true
+                },
+                "preview": {
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/documents.schema.json]: should not be here?

File: docs/schema-dumps/oggbundle/documents.schema.json [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+                        "privacy_layer_yes"
+                    ]
+                },
+                "document_author": {
+                    "type": "string",
+                    "title": "Autor",
+                    "description": "Nachname Vorname oder ein Benutzerk\u00fcrzel (wird automatisch nach Nachname Vorname aufgel\u00f6st).",
+                    "_zope_schema_type": "TextLine"
+                },
+                "archival_file_state": {
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/documents.schema.json]: should not be here?

File: docs/schema-dumps/oggbundle/documents.schema.json [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+                    "description": "",
+                    "_zope_schema_type": "NamedBlobFile"
+                },
+                "description": {
+                    "type": "string",
+                    "title": "Beschreibung",
+                    "description": "",
+                    "_zope_schema_type": "Text"
+                },
+                "archival_file": {
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/documents.schema.json]: should not be here?

File: docs/schema-dumps/oggbundle/documents.schema.json [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+                    "enum": [
+                        "unprotected",
+                        "confidential",
+                        "classified"
+                    ]
+                },
+                "parent_guid": {
+                    "type": "string"
+                },
+                "file": {
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/documents.schema.json]: there should be two fields here in our schema, `filename` and `path`

or 

we could always use `title` as filename, in that case `title` should be required?

File: docs/schema-dumps/oggbundle/dossiers.schema.json [hunk 7]
```diff
@@ -0,0 +1,319 @@
...
+                        "privacy_layer_yes"
+                    ]
+                },
+                "former_reference_number": {
+                    "type": "string",
+                    "title": "Fr\u00fcheres Aktenzeichen",
+                    "description": "",
+                    "_zope_schema_type": "TextLine"
+                },
+                "temporary_former_reference_number": {
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/dossiers.schema.json]: should not be here?

File: docs/schema-dumps/oggbundle/repofolders.schema.json [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+                    "default": "unchecked",
+                    "enum": [
+                        "unchecked",
+                        "public",
+                        "limited-public",
+                        "private"
+                    ]
+                },
+                "review_state": {
+                    "type": "string"
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/repofolders.schema.json]: enum?

File: docs/schema-dumps/oggbundle/documents.schema.json [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+                    "default": "unchecked",
+                    "enum": [
+                        "unchecked",
+                        "public",
+                        "limited-public",
+                        "private"
+                    ]
+                },
+                "review_state": {
+                    "type": "string"
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/documents.schema.json]: enum?

File: docs/schema-dumps/oggbundle/dossiers.schema.json [hunk 10]
```diff
@@ -0,0 +1,319 @@
...
+                    ]
+                },
+                "container_location": {
+                    "type": "string",
+                    "title": "Beh\u00e4ltnis-Standort",
+                    "description": "Standortangabe des Beh\u00e4lters, in dem ein Dossier in Papierform abgelegt ist",
+                    "_zope_schema_type": "TextLine"
+                },
+                "review_state": {
+                    "type": "string"
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/dossiers.schema.json]: enum?

File: docs/schema-dumps/oggbundle/reporoots.schema.json [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+                "version": {
+                    "type": "string",
+                    "title": "Version",
+                    "description": "",
+                    "_zope_schema_type": "TextLine"
+                },
+                "_permissions": {
+                    "$ref": "#/definitions/permission"
+                },
+                "review_state": {
```
Comment (deiferni) [inline docs/schema-dumps/oggbundle/reporoots.schema.json]: enum?

File: docs/schema-dumps/oggbundle/documents.schema.json (hunk 6 above)
Comment (lukasgraf) [inline docs/schema-dumps/oggbundle/documents.schema.json]: 👍  for `filename` and  `title`

File: docs/schema-dumps/oggbundle/documents.schema.json (hunk 9 above)
Comment (lukasgraf) [inline docs/schema-dumps/oggbundle/documents.schema.json]: Planned as a future enhancement, yes :trollface:

File: docs/schema-dumps/oggbundle/documents.schema.json (hunk 5 above)
Comment (lukasgraf) [inline docs/schema-dumps/oggbundle/documents.schema.json]: Same as `archival_file_status` - doesn't hurt, though not needed right now

**Comment by lukasgraf** [comment]
@deiferni updated

**Comment by lukasgraf** [comment]
@phgross updated

**Pull request merged**